home_style.css

```diff
@@ -0,0 +1,46 @@
+.title, h2{
+color:ghostwhite;
+}
+
+h1{
+    font-size: 90px;
+    text-align:end;
+}
+
+
+
+.container {
+    position: relative;
+    text-align: left;
+    color: white;
+    margin-top: 50px;
+}
+
+
+body{
+    background-color: #659dbd;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+}
+
+ /*  Text to right */
+.rest{
+    float:right;
```

Comment:
Float is not used anymore for layouts as we have amazing tools in our CSS toolkits such as flexbox and grid box which are more widely supported. Floats existed in a time before these tools were invented. Check out this article to get a better understanding https://css-tricks.com/is-css-float-deprecated/